src/main/java/com/javadiscord/javabot/other/Embeds.java

```diff
@@ -147,25 +90,20 @@ public static MessageEmbed configEmbed (Object ev, String title, String descript
                 else eb.setDescription(description);
 
                 eb.setFooter(user.getAsTag(), user.getEffectiveAvatarUrl())
-                .setTimestamp(new Date().toInstant());
+                .setTimestamp(Instant.now());
 
         return eb.build();
     }
 
-    public static MessageEmbed configEmbed (Object ev, String title, String description, String imageLink, String value, boolean showValue, boolean channel) {
-
+    public static MessageEmbed configEmbed(Object ev, String title, String description, String imageLink, String value, boolean showValue, boolean channel) {
         return Embeds.configEmbed(ev, title, description, imageLink, value, showValue, channel, false);
     }
-
```

Comment:
I think there should still be one line separating each method, so don't remove this one.

Reply:
sorry, that was a mistake, didn't see it